spec/models/registration_spec.rb

```diff
@@ -661,4 +661,14 @@
     expect(reg).not_to be_valid
     expect(reg.errors[:registered_at]).to include("can't be blank")
   end
+
+  describe '#entry_fee_with_donation' do
+    it 'returns a hash of entry fee and currency code' do
```

Comment:
It doesn't return a `hash` though?!

Reply:
Before I thought of using `Money` - fixed!